orchestrator/orchestrator.go

```diff
@@ -161,6 +161,21 @@ func (s *Orchestrator) executeNode(wf *process.Process, n *process.Process_Node,
 	return nil
 }
 
+// filterMatch returns true if the data match the current list of filters.
+func (s *Orchestrator) filterMatch(f *process.Process_Node_Filter, wf *process.Process, nodeKey string, exec *execution.Execution, data *types.Struct) bool {
+	for _, condition := range f.Conditions {
+		resolvedData, err := s.resolveRef(wf, exec, nodeKey, data, condition.Ref)
+		if err != nil {
+			return false
```

Comment:
We should return this error to the user
```suggestion
                         s.ErrC <- err
			return false
```